backend/de.metas.business/src/main/java/de/metas/costing/impl/CurrentCostsRepository.java

```diff
@@ -423,4 +424,16 @@ public void updateCostRecord(
 		saveRecord(costRecord);
 	}
 
+	public boolean hasCostsInCurrency(@NonNull final I_C_AcctSchema acctSchema, @NonNull final CurrencyId currencyId)
+	{
+		return queryBL.createQueryBuilder(I_M_Cost.class)
+				.addOnlyActiveRecordsFilter()
+				.addOnlyContextClient()
```

Comment:
pls avoid using `addOnlyContextClient`.
In this case you really don't have to because u are filtering by AD_Org_ID anyways

Reply:
done.